firefox_puppeteer/api/software_update.py

```diff
@@ -0,0 +1,499 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import re
+
+import mozinfo
+
+from ..base import BaseLib
+from .appinfo import AppInfo
+from .prefs import Preferences
+
+
+class ActiveUpdate(BaseLib):
+
+    def __getattr__(self, attr):
+        value = self.marionette.execute_script("""
+          let ums = Cc['@mozilla.org/updates/update-manager;1']
```

Comment:
> Sorry. This is the first time I've ever worked with any of this Firefox
> API stuff, so I'm not really sure what I'm doing. Hopefully |Cc| is the
> last one that needs to be changed.

No worries Bob! You are doing great on all that. Yes, `Cc` was the last
shotcut which needed to be ripped off. Thanks.